booking/src/main/java/org/dcsa/conformance/standards/booking/checks/BookingChecks.java

```diff
@@ -372,24 +407,9 @@ private static Consumer<MultiAttributeValidator> allDg(Consumer<MultiAttributeVa
         if (body.get("confirmedEquipments") == null) {
           issues.add("confirmedEquipments for confirmed booking is not present");
         }
-        if (body.get("termsAndConditions") == null) {
-          issues.add("termsAndConditions for confirmed booking is not present");
-        }
         if (body.get("transportPlan") == null) {
           issues.add("transportPlan for confirmed booking is not present");
         }
-        if (body.get("carrierClauses") == null) {
-          issues.add("carrierClauses for confirmed booking is not present");
-        }
-        if (body.get("charges") == null) {
-          issues.add("charges for confirmed booking is not present");
-        }
-        if (body.get("shipmentCutOffTimes") == null) {
-          issues.add("shipmentCutOffTimes for confirmed booking is not present");
-        }
```

Comment:
@HenrikHL: I did not see you challenging this one being present on a confirmed booking, so I assume `shipmentCutOffTimes` is still required on a confirmed booking (and therefore this should not be removed).